src/nblint.py

```diff
@@ -9,6 +9,8 @@ def main():
     dir = os.environ["INPUT_DIRECTORY"]
     ignore_paths = os.environ["INPUT_IGNORE_PATHS"]
     ignore_paths = ignore_paths.split(",")
+    if "" in ignore_paths:
+        ignore_paths.remove("")
```

Comment:
Btw, `list.remove` only removes the first instance of the item. Seems okay if you want to handle `ignore_paths == ""` and `ignore_paths.endswith(",")`, though it won't properly handle extra commas, like `ignore_paths == "foo,,bar,baz,"`. (People shouldn't be writing those anyways.)

Reply:
Ah right I overlooked that, but I think it should be fine to leave for now (as you mentioned people should hopefully be avoiding those)

Reply:
Yep, agree